src/tig.c

```diff
@@ -557,7 +557,7 @@ parse_options(int argc, const char *argv[], bool pager_mode)
 				printf("tig version %s\n", TIG_VERSION);
 #ifdef NCURSES_VERSION
 				printf("%s version %s.%d\n",
-#ifdef NCURSES_WIDECHAR
+#ifdef HAVE_NCURSESW_CURSES_H
```

Comment:
Thanks for reporting and fixing the problem. To be consistent with `ax_with_curses.m4` we probably have to make it `#if defined(HAVE_NCURSESW_CURSES_H) || defined(HAVE_NCURSESW_H)`, don't you think ?

Reply:
I believe you're correct and I've force-pushed an update that uses both. I'm not sure how I missed that.